src/Moryx.Products.Management/Moryx.Products.Management.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net45;net5.0</TargetFrameworks>
```

Comment:
Instead of only adding net5.0 also the minimal target framework "netstandard2.0" should be available

Reply:
I added target framework netstandard2.1 due to restrictions through Moryx.Model. This package does not support netstandard2.0.